### PR TITLE
Japanese support

### PR DESCRIPTION
**Pull request by Sergeanur**

### CODE REVIEW

File: src/core/Game.cpp [hunk 1]
```diff
@@ -97,8 +97,8 @@ bool CGame::noProstitutes;
 bool CGame::playingIntro;
 char CGame::aDatFile[32];
 #ifdef MORE_LANGUAGES
-bool CGame::polishGame = false;
```
Comment (erorcun) [inline src/core/Game.cpp:100]: no polish? :c

**Comment by Noskillx** [comment]
Imo bring Polish back and then can be merged.

File: src/core/Game.cpp (hunk 1 above)
Comment (Sergeanur) [reply src/core/Game.cpp:100]: Unused variable

Comment (erorcun) [reply src/core/Game.cpp:100]: oh ok, lgtm then

**Pull request merged**